### PR TITLE
Fixed Quarkus health paths

### PR DESCRIPTION
**Pull request by dloiacono**
## Description
The way the health path is builded for quarkus application is not correct.

## Type of change
 - [X] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [X] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [X] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [x] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [X] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [X] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by rohanKanojia** [comment]
I think you would need to modify tests too with respect to this change:
```
[ERROR]   QuarkusHealthCheckEnricherTest.createWithCustomPathInKubernetes:105 [Extracted: spec] 
Expecting:
  <[("HTTP", "/my-custom-path/live", "HTTP", "/my-custom-path/ready")]>
to contain exactly (and in same order):
  <[("HTTP", "/my-custom-path", "HTTP", "/my-custom-path")]>
but some elements were not found:
  <[("HTTP", "/my-custom-path", "HTTP", "/my-custom-path")]>
and others were not expected:
  <[("HTTP", "/my-custom-path/live", "HTTP", "/my-custom-path/ready")]>

[ERROR]   QuarkusHealthCheckEnricherTest.createWithDefaultsInKubernetes:86 [Extracted: spec] 
Expecting:
  <[("HTTP", "/health/live", "HTTP", "/health/ready")]>
to contain exactly (and in same order):
  <[("HTTP", "/health", "HTTP", "/health")]>
but some elements were not found:
  <[("HTTP", "/health", "HTTP", "/health")]>
and others were not expected:
  <[("HTTP", "/health/live", "HTTP", "/health/ready")]>
```

**Comment by manusa** [comment]
Related:
 - https://quarkus.io/guides/microprofile-health#architecture
 - https://github.com/eclipse/jkube/issues/443
 - https://github.com/eclipse/jkube/issues/450#issuecomment-726750845

File: jkube-kit/jkube-kit-quarkus/src/main/java/org/eclipse/jkube/quarkus/enricher/QuarkusHealthCheckEnricher.java [hunk 1]
```diff
@@ -29,6 +29,9 @@
  */
 public class QuarkusHealthCheckEnricher extends AbstractHealthCheckEnricher {
 
+    public static final String READY_SUBPATH = "/ready";
+    public static final String LIVE_SUBPATH = "/live";
```
Comment (manusa) [inline jkube-kit/jkube-kit-quarkus/src/main/java/org/eclipse/jkube/quarkus/enricher/QuarkusHealthCheckEnricher.java:33]: These can/should be private

**Comment by manusa** [comment]
Hi @dloiacono 
We want to release 1.1.1 early next week.
If you [want this to be part of that release](https://gitter.im/eclipse/jkube?at=602d4e7493539e23437fe843), please create the commit on top of the 1.1.x branch.
On the other hand, if you fine with this being part of 1.2.0 (release date TBD), just let us know.

**Pull request closed without merging**